go/vt/sqlparser/keywords.go

```diff
@@ -301,6 +301,7 @@ var keywords = []keyword{
 	{"insensitive", UNUSED},
 	{"insert", INSERT},
 	{"insert_method", INSERT_METHOD},
+	{"invisible", INVISIBLE},
```

Comment:
It looks like this keyword and `visible` as well are already defined in the parser so they don't need to be updated there.